dpbench/runner.py

```diff
@@ -30,6 +30,8 @@ def run_benchmark(
     bdir = "benchmarks/" + bname
     if not pkg_resources.resource_isdir(__name__, bdir):
         return
+    if bname == "__pycache__":
```

Comment:
Ok for now, but we should convert benchmark into a module and stop using pkg_resources.